app/LoadedApp/LoadedApp.tsx

```diff
@@ -175,11 +204,14 @@ type LoadedAppClassProps = {
   language: 'en' | 'es';
   currency: 'USD' | '';
   server: string;
+  sendAll: boolean;
+  background: backgroundType;
 };
 
 class LoadedAppClass extends Component<LoadedAppClassProps, AppStateLoaded> {
   rpc: RPC;
   dim: EmitterSubscription;
+  sta: NativeEventSubscription;
```

Comment:
Not a blocking concern: If I had to guess, sta is "state" but it's not immediately obvious to me. Could we call it 'state'?
```suggestion
  state: NativeEventSubscription;
```